dask_cuda/cli.py

```diff
@@ -404,9 +423,52 @@ async def run():
             logger.info("End worker")
 
 
-def go():
-    main()
+@cuda.command(name="config", context_settings=dict(ignore_unknown_options=True))
+@scheduler
+@preload_argv
+@scheduler_file
+@click.option(
+    "--get-cluster-configuration",
+    "get_cluster_conf",
+    default=False,
+    is_flag=True,
+    required=False,
+    show_default=True,
+    help="""Print a table of the current cluster configuration""",
+)
```

Comment:
I guess this is the only argument and without it nothing will happen. Does it make more sense to remove and make the default action (without any non-mandatory arguments) print the configuration?

Reply:
Yeah was thinking that as well, just wasn't sure if it made more sense to do in a separate PR since the diff here is fairly large already.

(cc @quasiben in case there's something non-obvious this command does when `--get-cluster-configuration` isn't passed as an option)